test/e2e/cluster.sh

```diff
@@ -42,6 +42,23 @@ ${ADD_NODE_POOL_INPUT}
 END
 fi
 
+if [ -n "$PRIVATE_SSH_KEY_FILE" ]; then
+  PRIVATE_SSH_KEY_FILE=$(realpath --relative-to=$(pwd) ${PRIVATE_SSH_KEY_FILE})
+fi
+
+function tryExit {
+  if [ "${AZURE_ENV}" != "AzureStackCloud" ]; then
+    exit 1
+  fi
+}
+
+function renameResultsFile {
+  JUNIT_PATH=$(pwd)/test/e2e/kubernetes/junit.xml
+  if [ "${AZURE_ENV}" == "AzureStackCloud" ] && [ -f ${JUNIT_PATH} ]; then
+    sudo mv ${JUNIT_PATH} $(pwd)/test/e2e/kubernetes/${1}-junit.xml
```

Comment:
is `sudo` needed?

Reply:
From the test runs this is needed otherwise there will be permission issue, since we are running the cluster.sh as "azureuser".

Reply:
Why is sudo needed? Is it the directory permissions?

Reply:
it may be better to `chown` the aks-engine directory before `cluster.sh` is executed.

Reply:
Tested again and it works without "sudo". Sorry for the false statement, I have removed the "sudo" part.